src/states/adult/mod.rs

```diff
@@ -6,6 +6,9 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
+#[cfg(all(test, feature = "mock_base"))]
```

Comment:
Why do we build it in mock_base case?

Reply:
Those tests don't require mock parsec because they don't use parsec at all.